gallery/visualtests/nxTextInput.js

```diff
@@ -134,11 +134,11 @@ describe('NxTextInput', function() {
   });
 
   describe('Long NxTextInput', function() {
-    it('looks right', simpleTest(textareaComponentSelector));
+    it('looks right', simpleTest(longComponentSelector));
```

Comment:
These selectors were just straight up wrong, and I happened to notice due to the applitools diffs that came from the selenium upgrade.